src/components/BareMetalHosts/StatusComponents.js

```diff
@@ -68,8 +68,12 @@ ValidationError.propTypes = {
   errorMessage: PropTypes.string.isRequired,
 };
 
-export const AddDiscoveredHostLink = host => (
-  <a>
+export const AddDiscoveredHostLink = ({ addHostProps }) => (
+  <a onClick={addHostProps.onClick}>
```

Comment:
Are we okay with showing the dialog straight away when clicking on the link ?

Maybe an effort in #427 could be relevant to this.

@andybraren @matthewcarleton @yaacov

Reply:
Definitely relevant. [We have a PR](https://github.com/openshift/openshift-origin-design/pull/179) introducing this new Status Popover pattern in the OpenShift Design repo and it seems likely to get merged.

Here's what a status popover could look like for a discovered host that includes an "Add host" action in the footer. We're still figuring out how discovered hosts should be sorted/separated to make sure they always appear at the top, but if the popover in #427 is easy to implement then something like the below would still be an improvement.

![add-host-popover](https://user-images.githubusercontent.com/9122899/58065263-1cf79380-7b53-11e9-8fa6-f2be2822a0b2.png)

Reply:
This should be easy to implement -- let me change the PR to add the popover. I really like it! :)

Reply:
This is now implemented in this PR